common/flatpak-error.h

```diff
@@ -42,6 +42,8 @@ G_BEGIN_DECLS
  * @FLATPAK_ERROR_NEED_NEW_FLATPAK: The App/Runtime needs a more recent version of flatpak.
  * @FLATPAK_ERROR_REMOTE_NOT_FOUND: The specified remote was not found.
  * @FLATPAK_ERROR_RUNTIME_NOT_FOUND: An runtime needed for the app was not found.
+ * @FLATPAK_ERROR_DOWNGRADE: The pulled commit is a downgrade, and a downgrade wasn't
+ *                           specifically allowed.
```

Comment:
Add `(Since: 1.0)` (or whatever version this is going to be released in). i.e. `The pulled commit … specifically allowed. (Since: 1.0)`

Reply:
Pushed a fixup